staging/src/k8s.io/apiextensions-apiserver/pkg/controller/status/naming_controller.go

```diff
@@ -191,26 +191,26 @@ func (c *NamingConditionController) calculateNamesAndConditions(in *apiextension
 		namesAcceptedCondition.Message = "no conflicts found"
 	}
 
-	// set EstablishedCondition to true if all names are accepted. Never set it back to false.
-	establishedCondition := apiextensions.CustomResourceDefinitionCondition{
-		Type:    apiextensions.Established,
+	// set InstallingCondition to true if all names are accepted. Never set it back to false.
+	installingCondition := apiextensions.CustomResourceDefinitionCondition{
+		Type:    apiextensions.Installing,
 		Status:  apiextensions.ConditionFalse,
 		Reason:  "NotAccepted",
 		Message: "not all names are accepted",
 	}
-	if old := apiextensions.FindCRDCondition(in, apiextensions.Established); old != nil {
-		establishedCondition = *old
+	if old := apiextensions.FindCRDCondition(in, apiextensions.Installing); old != nil {
+		installingCondition = *old
 	}
-	if establishedCondition.Status != apiextensions.ConditionTrue && namesAcceptedCondition.Status == apiextensions.ConditionTrue {
-		establishedCondition = apiextensions.CustomResourceDefinitionCondition{
-			Type:    apiextensions.Established,
+	if installingCondition.Status != apiextensions.ConditionTrue && namesAcceptedCondition.Status == apiextensions.ConditionTrue {
+		installingCondition = apiextensions.CustomResourceDefinitionCondition{
+			Type:    apiextensions.Installing,
 			Status:  apiextensions.ConditionTrue,
 			Reason:  "InitialNamesAccepted",
```

Comment:
the Reason and Message is actually misleading if the resource is already established. It won't be the initial names anymore.